cmd/minishift/cmd/start_preflight.go

```diff
@@ -311,9 +313,8 @@ func checkStorageMounted(driver drivers.Driver) bool {
 // space available.
 func checkStorageUsage(driver drivers.Driver) bool {
 	usedPercentage := getDiskUsage(driver, StorageDisk)
-	fmt.Printf("%s ", usedPercentage)
-	usedPercentage = strings.TrimRight(usedPercentage, "%")
-	usage, err := strconv.ParseInt(usedPercentage, 10, 8)
+	fmt.Printf("%s used ", usedPercentage)
+	usage, err := strconv.ParseInt(stringUtils.GetOnlyNumbers(usedPercentage), 10, 8)
```

Comment:
Not related to this PR, but commenting because we are at it. 
@gbraad AFAICT we are not gaining anything by using int bit size as `8` in strconv.ParseInt("string", 10, `8`) because it returns int64 (for a 64bit system) anyway. So we are not saving any memory here.  So I will suggest we replace `ParseInt` with `Atoi` and it would make code simple by a tiny bit. WDYT?

Reply:
I think `strconv.Atoi(stringUtils.GetOnlyNumbers(usedPercentage))` doesn't read very nicely, but we can change...